src/main/java/com/project/parkinglot/controller/ParkingAreaController.java

```diff
@@ -56,4 +58,16 @@ public CustomResponse<String> deleteParkingAreaById(@PathVariable("parkingAreaId
 
     }
 
+    @PutMapping("/{parkingAreaId}")
+    @PreAuthorize("hasAuthority('ROLE_ADMIN')")
+    public CustomResponse<String> updateParkingArea(
+            @PathVariable("parkingAreaId") @UUID final String parkingAreaId,
+            @RequestBody @Valid final ParkingAreaUpdateRequest parkingAreaUpdateRequest
+            ){
+        final ParkingArea parkingArea = parkingAreaUpdateService
+                .parkingAreaUpdateByCapacity(parkingAreaId,parkingAreaUpdateRequest);
+
+        return CustomResponse.ok("Parking area with id " + parkingArea.getId() + " is updated");
```

Comment:
bu kısım frontend'in daha kolay işlemesi için sadece id değerrini dönmesi daha mı uygun olur @mehmetseymusyuzen @Rapter1990 @moaydogdu @anilhazar